### PR TITLE
Ana/fix arrow links between proposals and improve styles

### LINKED ISSUE #3938: Single proposal links broken (again)
I thought I had fixed this 🤦

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #3938

**Description:**

Simple PR to fix a bug inherited from our big routing shift. Yes, there are still bugs from this xD

Fixes this and adds better styling (here I was hovering on the arrow):

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
unit tests fail

**Comment by Bitcoinera** [comment]
Let's merge! 💃

File: src/components/governance/PageProposal.vue [hunk 1]
```diff
@@ -10,21 +10,27 @@
...
               :style="{ visibility: getPrevProposalId ? 'visible' : 'hidden' }"
-              class="read-more-link"
             >
               <i class="material-icons notranslate">chevron_left</i>
-            </router-link>
+            </div>
             <h2 class="proposal-title">{{ proposal.title }}</h2>
-            <router-link
-              :to="`/proposals/` + getNextProposalId"
+            <div class="read-more-link"
```
Comment (faboweb) [inline src/components/governance/PageProposal.vue:24]: why was this change necessary?

Comment (faboweb) [inline src/components/governance/PageProposal.vue:24]: why was this change necessary?

Comment (Bitcoinera) [reply src/components/governance/PageProposal.vue:24]: Yes, you can also do:

```
<router-link class="read-more-link" :to="{name: 'Proposal',
                  params: { proposalId: String(getPrevProposalId) }}" 
            :style="{ visibility: getPrevProposalId ? 'visible' : 'hidden' }">
              <i class="material-icons notranslate">chevron_left</i>
            </router-link>
```

But in other places we are using `$router.push` already for this, so I was trying to be consistent

**Comment by jbibla** [comment]
🦨i think we should remove the arrows and left-align the proposal title. 🦨

**Comment by Bitcoinera** [comment]
> 🦨i think we should remove the arrows and left-align the proposal title. 🦨

I'm still waiting on clarification on this one 😄 where you possessed maybe by the skunk when you wrote this?

I think the arrows are pretty neat.

Let's merge! 💃

**Comment by jbibla** [comment]
ana and i discussed — we agreed to leave this for now and move to a more consistent button style in the future

**Pull request merged**